apisix/control/v1.lua

```diff
@@ -189,4 +212,10 @@ return {
         uris = {"/gc"},
         handler = _M.trigger_gc,
     },
+    -- /v1/dump
```

Comment:
Please use restful like API names for dump specific resources, like `/v1/upstreams/xxx` and `/v1/upstreams`.